support/windows-server/system-management-components/PowerShell Console characters garbled for CKJ languages

```diff
@@ -0,0 +1,41 @@
+---
+title: PowerShell Console characters garbled for Chinese, Japanese and Korean languages on Windows Server 2022
+description: PowerShell Console characters garbled for Chinese, Japanese and Korean languages on Windows Server 2022.
+ms.date: 08/13/2021
+author: kryalava
+ms.author: kryalava 
+manager: dcscontentpm
+audience: itpro
+ms.topic: troubleshooting
+ms.prod: windows-server
+localization_priority: medium
+ms.reviewer: slee
+ms.prod-support-area-path: PowerShell
+ms.technology: windows-server-system-management-components
+---
+# PowerShell console characters are garbled for Chinese, Japanese and Korean languages on Windows Server 2022
+
+This article provides a resolution to an issue that characters in PowerShell console getting garbled.
+
+_Applies to:_ &nbsp; Windows Server 2022
+_Original KB number:_ &nbsp;
+
+## Symptom
+
+When you launch PowerShell console the characters are getting garbbled. This applies only to Chinese, Japanese and Korean languages SKUs.
+
+## Resolution
+
+To fix the issue, there are 3 workarounds listed below. Either of them can be used.
+
+## Workaround 1: Launch the Powershell from cmd.exe
+
+Start cmd.exe and run powershell.exe from within the cmd console. This opens up a PowerShell console without any font issues.
+
+## Workaround 2: Launch the Powershell with this command
+
+Start PowerShell with the command `Powershell -noni` from the search bar. This opens up a PowerShell console without any font issues.
```

Comment:
Starting PowerShell with the `-noninteractive` option is not a real solution. When you do that there are certain features that will not be available. For example, the PSReadLine module will not load so many of the command line editing features will be missing. This is not a recommended solution.

It also tells me that the problem may be in PSReadLine. This should be reported and investigated.

Reply:
I can remove this option.